src/front/spv/mod.rs

```diff
@@ -518,6 +519,12 @@ impl<I: Iterator<Item = u32>> Parser<I> {
             log::debug!("\t\t{:?} [{}]", inst.op, inst.wc);
 
             match inst.op {
+                Op::Undef => {
+                    inst.expect(3)?;
+                    let _result_type_id = self.next()?;
+                    let _result_id = self.next()?;
+                    //TODO?
```

Comment:
Nit: as we are not doing anything with this (yet), maybe better to not initialize the variable? I do not know if the rust compiler removes the variable declarations when prefixed with a `_`, otherwise I suggest:
```suggestion
                    //TODO When an intermediate object is undefined, set it to undefined?
                    self.next()?;
                    self.next()?;                    
```
If I understand your TODO correctly, we want to mark the said intermediate object as undefined?

Reply:
We know what these are (i.e. result ID), so erasing this information is also not helpful if we are meant to come back to this.